pkg/machine/qemu/options_darwin_arm64.go

```diff
@@ -34,3 +35,23 @@ func (v *MachineVM) archRemovalFiles() []string {
 func getOvmfDir(imagePath, vmName string) string {
 	return filepath.Join(filepath.Dir(imagePath), vmName+"_ovmf_vars.fd")
 }
+
+/*
+ *  QEmu can be installed in multiple locations on MacOS, especially on 
+ *  Apple Silicon systems.  A build from source will likely install it in
+ *  /usr/local/bin, whereas Homebrew package management standard is to
+ *  install in /opt/homebrew
+ */
+func getEdk2CodeFd(name string) string {
+	dirs := []string{
+		"/usr/local/share/qemu",
+		"/opt/homebrew/share/qemu",
+	}
+	for _, dir := range dirs {
+		fullpath := filepath.Join(dir, name)
+		if _, err := os.Stat(fullpath); err == nil {
+			return fullpath
+		}
+	}
+	return name
```

Comment:
In case if nothing is found inside loop or `stat` call fails, it should default to `return filepath.Join(dirs[0], name)` instead of `return name` at line no 56. Otherwise default return would be `edk2-aarch64-code.fd` which is not a valid path.

Reply:
I followed the convention established in [options_linux_arm64.go](https://github.com/containers/podman/blob/32a493df0c488b793c307a060016dd7ea050dbff/pkg/machine/qemu/options_linux_arm64.go#L40).  I believe the rationale is to return a naked filename to pass to `qemu` and let it deal with trying to find it or erroring out.  I considered adding go-style `error` returns across the package but decided instead to follow already-established patterns.